app/src/org/commcare/utils/ConsumerAppsUtil.java

```diff
@@ -1,53 +1,12 @@
 package org.commcare.utils;
 
-import org.commcare.CommCareApplication;
-import org.commcare.activities.CommCareHomeActivity;
-import org.commcare.engine.resource.installers.SingleAppInstallation;
 import org.commcare.views.dialogs.CustomProgressDialog;
-import org.javarosa.core.model.User;
-import org.javarosa.core.reference.InvalidReferenceException;
-import org.javarosa.core.reference.ReferenceManager;
-import org.javarosa.xml.ElementParser;
-import org.kxml2.io.KXmlParser;
-import org.xmlpull.v1.XmlPullParserException;
-
-import java.io.IOException;
-import java.io.InputStream;
 
 /**
  * Created by amstone326 on 5/12/16.
  */
 public class ConsumerAppsUtil {
 
-    public static void checkForChangedLocalRestoreFile(CommCareHomeActivity context) {
```

Comment:
Hey yea that's fine. I left it because it's likely to get used again at some point, but I can just add it back in.